src/mongocxx/database.hpp

```diff
@@ -248,6 +248,8 @@ class MONGOCXX_API database {
 
     class MONGOCXX_PRIVATE impl;
     std::unique_ptr<impl> _impl;
+    const impl& get_impl() const;
```

Comment:
Private methods should start with _. Also, private members should have MONGOCXX_PRIVATE on them in the declaration.